.github/workflows/pack-upload.yml

```diff
@@ -105,25 +105,25 @@ jobs:
           yarn oclif upload tarballs
           ./scripts/upload/deb
 
-  # pack-and-upload-windows:
-  #   runs-on: ubuntu-latest
-  #   env:
-  #     AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-  #     AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-  #   steps:
-  #     - uses: actions/checkout@v3
-  #     - uses: actions/setup-node@v3
-  #       with:
-  #         node-version: lts/*
-  #         cache: yarn
-  #     - name: Set up Homebrew
-  #       id: set-up-homebrew
-  #       uses: Homebrew/actions/setup-homebrew@41775cf0c82ef066f1eb39cea1bd74697ca5b735
-  #     - name: Install NSIS
-  #       run: brew install nsis
-  #     - name: yarn install
-  #       run: yarn --immutable --network-timeout 1000000
-  #     - name: pack windows installer
-  #       run: yarn pack:win
-  #     - name: upload windows installer
-  #       run: yarn upload:win
+  pack-and-upload-windows:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
+          cache: yarn
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@41775cf0c82ef066f1eb39cea1bd74697ca5b735
+      - name: Install NSIS
+        run: brew install nsis
+      - name: yarn install
+        run: yarn --immutable --network-timeout 1000000
+      - name: pack windows installer
+        run: yarn pack:win
+      - name: upload windows installer
+        run: yarn upload:win
```

Comment:
Where do these scripts come from? `pack:win` & `upload:win`
I would expect these to need a root flag used elsewhere `--root="./packages/cli"`

Reply:
The scripts are [here](https://github.com/heroku/cli/blob/da0d23a9c5ca47efc5655ebf3fd24ddb009274c7/packages/cli/package.json#L332-L341) in the cli package.json. And yes, you are right, they do need the root flag, thank you.

Reply:
wups, missed that when searching somehow. Do they run successfully without `--root="./packages/cli"`?

Reply:
Updated to include the root.